backend/sozisel/lib/sozisel_web/schema/resolvers/participant_resolvers.ex

```diff
@@ -0,0 +1,112 @@
+defmodule SoziselWeb.Schema.Resolvers.ParticipantResolvers do
+  alias Sozisel.Repo
+  alias SoziselWeb.Schema.Helpers
+  alias SoziselWeb.Schema.Subscriptions.Topics
+
+  alias Sozisel.Model.{
+    Sessions,
+    Events,
+    LaunchedEvents,
+    Participants,
+    Quizzes,
+    EventResults,
+    Utils
+  }
+
+  alias Events.Event
+  alias LaunchedEvents.LaunchedEvent
+  alias Sessions.Session
+  alias Participants.Participant
+  alias Quizzes.{QuizResult, ParticipantAnswer, TrackNode}
+
+  require Logger
+
+  # password is optional so we can't always pattern match on all arguments
+  def join_session(
+        _parent,
+        %{input: %{email: email, full_name: full_name, session_id: session_id} = input},
+        _ctx
+      ) do
+    entry_password = Map.get(input, :entry_password)
+
+    with %Session{entry_password: ^entry_password} <- Repo.get_by(Session, id: session_id),
```

Comment:
I think we should also check if session is active. Participant can't join session that has ended or not started yet.